.github/workflows/ruby_tests.yml

```diff
@@ -4,94 +4,26 @@ on:
   push:
     branches:
     - master
+
+
+concurrency:
+  group: ${{ github.ref_name }}
+  cancel-in-progress: true
+
 env:
   RAILS_ENV: test
   DATABASE_URL: postgresql://postgres:@localhost/test
   DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL: true
+
 jobs:
   rubocop:
-    runs-on: ubuntu-latest
-    env:
-      BUNDLE_FOREMAN: '0'
```

Comment:
This one needs to remain for the RuboCop action. At least with the current Gemfile. It's a very complex Gemfile and it's questionable if it's a good idea. I'd prefer if it was much closer to our template (https://github.com/theforeman/foreman_plugin_template/blob/master/Gemfile) and only listed what it actually needed (like RuboCop).

Reply:
@ekohl i didn't clearly get it, could you please elaborate further?

Reply:
If you look at https://github.com/theforeman/foreman_puppet/blob/master/Gemfile you can see it has different behavior depending on whether the environment variable `BUNDLE_FOREMAN` has a specific value. Since you're leaving this out, it flips to the other behavior, which causes the failures you see.

I was then asking myself if this behavior actually needs to remain, but to move ahead faster I'd start with setting the environment variable as it set in the current CI.

Reply:
Sounds good to me. Might be a good idea to change it soon, otherwise people will stumble across it again, eventually.